numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -4913,6 +4930,12 @@ initialize_static_globals(void)
         return -1;
     }
 
+    /* Initialize from certain environment variabels: */
+    char *env = getenv("NPY_NUMPY_2_BEHAVIOR");
+    if (env != NULL && strcmp(env, "0") != 0) {
+        npy_numpy2_behavior = NPY_TRUE;
+    }
+
```

Comment:
It seems coverage is not gathering statistics from the appropriate CI run? That may be OK for now but will be problematic going forward